c/busybox-1.22.0/date_false-no-overflow.i

```diff
@@ -3768,6 +3768,30 @@ unsigned int sleep(unsigned int sec) {
   __VERIFIER_assume(retval <= sec);
   return retval;
 }
+char *strptime(const char *s, const char *format, struct tm *result)
```

Comment:
Wouldn't that require a full-blown string parser? Any concrete ideas what to do instead?

Reply:
My opinion is that this is OK since it is an overapproximation of the real behavior.
I will have a look whether this will even make a difference in the corresponding tasks

Reply:
Yes, it would... I checked the code more closely and as far as I understand it, the code checks for some format of the string (starting with . and then possibly two numbers), but not for particular numbers, therefore the constraints put on the string should not have any effect on the numbers in the result. So I guess this implementation is ok after all.